Examples/instance-flow.md

```diff
@@ -1,61 +1,105 @@
 {{{
 "title": "Instance Flow",
-"date": "09-01-2016",
-"author": "",
+"date": "12-19-2018",
+"author": "Julio Castanar",
 "attachments": [],
-"contentIsHTML": false
+"contentIsHTML": false,
+"keywords": ["api", "example", "authentication", "deployment", "provider", "policy box", "box", "instance", "terminate" ]
 }}}
 
-### Using the API: from Registering a Provider to the Deployment of an Instance
+**In this article**
 
-This guide shows all the steps you need to deploy a box using the Cloud Application Manager API.
-* Authentication and Deployment Variables
-* Register a Provider
-* Create a New Deployment Policy Box
-* Create a Box
-* Deploy the Instance
-* Terminate the Instance
+* [Overview](#overview)
+* [Authenticate with Cloud Application Manager](#authenticate-with-cloud-application-manager)
+* [Obtain Deployment Variables](#obtain-deployment-variables)
+* [Register a Provider in Cloud Application Manager](#register-a-provider-in-cloud-application-manager)
+* [Create a Deployment Policy Box](#create-a-deployment-policy-box)
+* [Create a Custom Box](#create-a-custom-box)
+* [Deploy the Instance](#deploy-the-instance)
+* [Terminate and Delete the Instance](#terminate-and-delete-the-instance)
+* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
 
-**Note:** We use cURL commands to send HTTP requests to the API objects in JSON. JSON is the required format for all API requests and responses.
+## Overview
+
+This guide shows **how to use Cloud Application Manager API from Registering a Provider to the Deployment of an Instance**.
+
+
+**Notes:** 
+1. You will use cURL commands to send HTTP requests to the API objects in JSON. JSON is the required format for all API requests and responses.
+
+2. cURL commands in examples use -k parameter for convenience allowing curl to proceed and operate even for server connections considered insecure. Don't use it in production.
```

Comment:
_2. cURL is used with the `-k` parameter for convenience, allowing it to proceed..._